### PR TITLE
UnicodeBlockSerializer

### PR DESCRIPTION
**Pull request by Pr0methean**
New serializer for Character.UnicodeBlock, with test. Takes into account
that this class is effectively an enum.

### CODE REVIEW

**Comment by 3xp0n3nt** [comment]
@Pr0methean Thanks for the contribution. Perhaps next time you could do this work on a local branch, and once everything is working, squash, push, and pull request. The email spam for every one of these commits for all 31 watchers of this repository isn't very polite, thanks! 😉

**Comment by Pr0methean** [comment]
@3xp0n3nt Will do. Sorry, I didn't see your comment until after I'd finished, and I didn't realize I'd left the PR open after the tests had failed.

**Comment by 3xp0n3nt** [comment]
@Pr0methean No problem. 😺

**Comment by magro** [comment]
@Pr0methean thanks for the PR! I don't see any reason why this shouldn't go directly into kryo, as it's part of the jdk and not jvm/vendor specific. Can you please submit the PR against kryo?

File: src/test/java/de/javakaffee/kryoserializers/UnicodeBlockSerializerTest.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        kryo.register(UnicodeBlock.class, new UnicodeBlockSerializer());
+    }
+
+    @Test
+    public void testBasicRoundTrip() {
+        byte[] serialized = serialize(kryo, UnicodeBlock.UNIFIED_CANADIAN_ABORIGINAL_SYLLABICS);
+        try {
+        assertSame(deserialize(kryo, serialized, UnicodeBlock.class),
+                UnicodeBlock.UNIFIED_CANADIAN_ABORIGINAL_SYLLABICS);
+        } catch (NullPointerException e) {
```
Comment (magro) [inline src/test/java/de/javakaffee/kryoserializers/UnicodeBlockSerializerTest.java]: Can be removed now?

Comment (Pr0methean) [inline src/test/java/de/javakaffee/kryoserializers/UnicodeBlockSerializerTest.java]: Done: https://github.com/magro/kryo-serializers/pull/70/files#diff-f384965c1464e80c384a904b985b8b87

**Comment by Pr0methean** [comment]
The update to the README is https://github.com/magro/kryo-serializers/pull/70/files#diff-0a369498a5a8db3ac8fa606b544c9810. The debug code is removed from the test in https://github.com/magro/kryo-serializers/pull/70/files#diff-328d875a27bcefdefd6a0d692c6b7d66

**Comment by magro** [comment]
Thanks!

**Pull request merged**